components/authz-service/storage/errors.go

```diff
@@ -52,18 +52,6 @@ func (e *TxCommitError) Error() string {
 }
 
 // MissingFieldError occurs when a required field was not passed.
-type MissingFieldError struct {
```

Comment:
Only `CreatePolicy` was using this, so opted to remove it in the standardization process.